### ISSUE TITLE
get appenonly / save metrics (or create labeled metrics for all config params)

### ISSUE THREAD

**Issue by rayderua** [oliver006/redis_exporter#606]
**Describe the problem**
Is it possible to get metrics for appendonly and save config params? 
for example
```
redis_config_appenonly{} 0|1
redis_config_save{seconds="<seconds>"} <changes>
```
It would like to be able to catch the state when both are disabled.

Our problem is:
someone of our admins set `appendonly off` and `save ""` and apparently forgot about it
Yesterday server restarted, and redis startup with old dump (about 2 month old) :(
We were able to restore almost everything from a backup  (dump each 15 minutes), but would like to limit ourselves from similar mistakes in the future.

Perhaps you can make a metric with labels for the rest of the config parameters, like 
```
redis_config{param="<param_name>" value="<value>"} 1
```

**Comment by oliver006** [comment]
I like the idea of recording all the config settings (and make it optional to opt in, it would add quite a few metrics), let me take a stab at that.

**Comment by oliver006** [comment]
Added this as part of #607 - released as [1.35.0](https://github.com/oliver006/redis_exporter/releases/tag/v1.35.0)
